File: src/main/java/com/rtomyj/yugiohAPI/YugiohAPIMain.java
package com.rtomyj.yugiohAPI;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

/**
 * Contains main method.
 */
@SpringBootApplication
@EnableScheduling
public class YugiohAPIMain {


	/**
	 * Inits the program. Main method for program.
	 * @param args Java arguments
	 */
	public static void main(String[] args) {
		SpringApplication.run(YugiohAPIMain.class, args);
	}

}

File: src/main/java/com/rtomyj/yugiohAPI/YugiohAPIMain.kt
package com.rtomyj.yugiohAPI

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.scheduling.annotation.EnableScheduling
import org.springframework.scheduling.annotation.EnableAsync
import kotlin.jvm.JvmStatic
import org.springframework.boot.SpringApplication

/**
 * Contains main method.
 */
@SpringBootApplication
@EnableScheduling
@EnableAsync
class YugiohAPIMain


/**
 * Inits the program. Main method for program.
 * @param args Java arguments
 */
fun main(args: Array<String>) {
    SpringApplication.run(YugiohAPIMain::class.java)
}
